tests/unit/test_logger.py

```diff
@@ -1,79 +1,105 @@
-import os
```

Comment:
we've had issues with double logging in the past so may be good to add an assert in against that

Reply:
I've updated tests to ensure exact counts